public/app/features/serviceaccounts/ServiceAccountsListPage.tsx

```diff
@@ -84,6 +100,7 @@ const ServiceAccountsListPage = ({
                     <th>ID</th>
                     <th>Roles</th>
                     <th>Tokens</th>
+                    <th style={{ width: '34px' }} />
```

Comment:
We already have styles for this component defined in `getStyles`, so it's better to move this declaration there as well.